pom.xml

```diff
@@ -61,29 +60,29 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-aggregator</artifactId>
-            <version>2.0</version>
+            <version>2.6</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
```

Comment:
What are `script-security`, `mailer` and `symbol-annotation` used for?

Reply:
Just to satisfy compilation and upper bounds dependencies.  They will be removed in a future change that is ready in my repository.  

When the plugin bill of materials is used, those unexpected dependencies are no longer required.

Reply:
If you'd prefer to not have the surprising dependency temporarily, I can update this pull request with the plugin bill of materials so that the dependencies are easier to understand.  The plugin bill of materials change is a larger change because it also adds a declarative Pipeline test and creates a `clover` keyword to be used in Jenkins Pipeline.

Reply:
See https://github.com/jenkinsci/clover-plugin/compare/master...MarkEWaite:use-plugin-bom for the `use-plugin-bom` commit that is added onto this branch.  It simplifies the test dependencies, adds a declarative Pipeline test, and adds the `clover` symbol that enables [Pipeline syntax snippet generator](https://www.jenkins.io/doc/book/pipeline/getting-started/#snippet-generator).  

Unfortunately, the snippet generator seems to ignore some of the entered values from the clover plugin fields.  I don't understand why those field values are ignored, but using the snippet generator is still easier (at least for me) than entering the syntax interactively.

Reply:
> I can update this pull request with the plugin bill of materials so that the dependencies are easier to understand. The plugin bill of materials change is a larger change.

Let's keep changes small, we can have these dependencies as an intermediate step, especially that they're in the test scope.